pulpcore/cli/core/export.py

```diff
@@ -16,16 +18,34 @@
 from pulpcore.cli.core.context import PulpExportContext, PulpExporterContext
 
 _ = gettext.gettext
+REPO_HREF_PATTERN = r"^/pulp/api/v3/repositories/(?P<plugin>\w+)/(?P<resource_type>\w+)/"
```

Comment:
Why define this hear instead of using `PulpRepositoryContext.HREF_PATTERN`?

Reply:
forgot about that.